js/stores/message_store.js

```diff
@@ -22,18 +24,39 @@ var MessageStore = assign({}, EventEmitter.prototype, {
   all: function() {
     return _messages;
   },
+
```

Comment:
ya, i was gonna bring up the `local` thing.. i think eventually we can have some sort of auth, so that you are identified by id, and the nick is just a display name.. that would be the slack way (true accounts). or, the more irc way, would involve less persistence.. where you could maybe be authed only for the duration of the session, so you get like a temporary id and you can only edit the last message from your id, but you can start over in a new session and have no history. (i know you can have accounts in irc but the philosophy seems different.)